bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/OptionsConstraint.java

```diff
@@ -69,6 +69,16 @@ default boolean isEnforceEnum() {
      * @return the list of enum names
      * @since com.adobe.cq.forms.core.components.models.form 5.4.0
      */
-    @JsonProperty("enumNames")
+    @JsonIgnore
     TextContent[] getEnumNamesAsTextContent();
```

Comment:
Can you remove this function itself, I don't think anyone would be using it yet. Let's have one public facing API for this.

Reply:
we are using it in sightly to enable rich text, eg: https://github.com/adobe/aem-core-forms-components/blob/master/ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkboxgroup/v1/checkboxgroup/widget.html#L28

Reply:
Please use the new API, it does not make sense to have two API's for the same thing.

Reply:
got it, removed old APi